src/sentry/static/sentry/app/views/ruleEditor/index.jsx

```diff
@@ -125,7 +126,7 @@ const RuleEditor = createReactClass({
       endpoint += rule.id + '/';
     }
 
-    let loadingIndicator = IndicatorStore.add('Saving...');
+    addMessage(t('Saving...'));
```

Comment:
👍 
For future PRs: https://github.com/getsentry/sentry/pull/7273/files#diff-6ead2464adfb74f5bbb5c4d515a793feR30